### PR TITLE
[POC] Sqlalchemy POC

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: spanner/google/cloud/spanner/db_api/cursor.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    def __init__(self, connection: Connection):
+        self.connection = connection
+        self.description = None
+        # Per PEP 249: The attribute is -1 in case no .execute*() has been
+        # performed on the cursor or the rowcount of the last operation
+        # cannot be determined by the interface.
+        self.rowcount = -1
+        # Per PEP 249: The arraysize attribute defaults to 1, meaning to fetch
+        # a single row at a time.
+        self.arraysize = 1
```
Comment (tswast) [inline spanner/google/cloud/spanner/db_api/cursor.py]: I regret actually following the PEP for this case. In most cases (in BigQuery at least), they'll get the best performance if the default is `None`.

File: docs/start_doc_server.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#
+g#
```
[the remark this answers is not in the document]
Comment (potiuk) [reply docs/start_doc_server.sh]: Typo. I have not been to careful with this WIP commit . :)

File: spanner/AUTHORS [hunk 3]
```diff
@@ -0,0 +1 @@
+SQLAlchemy spanner dialect was created by Jarek Potiu.
```
[the remark this answers is not in the document]
Comment (potiuk) [reply spanner/AUTHORS]: Sure. It's quite far from being correct yet :). Probably there are more typos like that, I would rather not focus on those just yet. But thanks for being interested :) 👍

File: spanner/google/cloud/spanner/db_api/cursor.py (hunk 1 above)
Comment (potiuk) [reply spanner/google/cloud/spanner/db_api/cursor.py]: Did what you suggested in 6284cdaf9ca6d1a241f625cb5071b27ee6af0770

File: docs/start_doc_server.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (potiuk) [reply docs/start_doc_server.sh]: Fixed in e7c8282bba87a3899797c1c925e7b7bc79a10a3d

File: spanner/AUTHORS (hunk 3 above)
[the remark this answers is not in the document]
Comment (potiuk) [reply spanner/AUTHORS]: Fixed in 4c533c8a1ff3d4f89a995b4f0a9e1b80bbbd9c71

**Comment by nuclearpinguin** [comment]
@potiuk should we close the PR?

**Pull request closed without merging**